From 04f7da2f190ddc636a5c268f2428f36517f92e25 Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Fri, 19 Apr 2019 02:24:04 +0200
Subject: [PATCH] configure.ac: Search for pkg-config with its
 PKG_PROG_PKG_CONFIG macro

Otherwise the pkg-config command might not be found and thus some of the
required packages (like ncurses) as well.

Signed-off-by: Lars Wendler <polynomial-c@gentoo.org>
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 5f9f9e78..711cb5df 100644
--- a/configure.ac
+++ b/configure.ac
@@ -27,6 +27,8 @@ AC_CANONICAL_HOST
 AC_CONFIG_HEADERS(config.h)
 AC_CONFIG_MACRO_DIR([m4])
 
+PKG_PROG_PKG_CONFIG
+
 # Enable silent build rules by default (Automake v1.11 or later).
 # Disable by either passing --disable-silent-rules to configure or passing V=1 to make
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])],
-- 
2.21.0

